### DISCUSSION TITLE
Port expander MCP23017 16bit option

### DISCUSSION THREAD

**Discussion by azgael** [shanteacontrols/OpenDeck#196] [Ideas]
Dear Paradajz
First of all thank you very much for this outstanding project, I was able with your documentation and very good instructions put a controller based on native pin of the Pico but with some customization on the pin use. (basically only the two I2C pins are unconnected)
Very very strait forward, I cannot stop being thank full for this!!!
Additionally I would like to increase the number of rotary switch (eventually ADC's also at later stage maybe ADS1115 or MAX11617 or TLV0838) but using a MCP23017 16bit I2C port expander that I have around from my Arduino playground. Do you think would make sense to include the option of a Port expander MCP23017 16bit  on future released, this way the board would be flexible to switch inputs to led outputs and vise versa instead of having Inputs or Outputs fixed by use of shift registers?

Thank you very much once again!!
Best regards, 
Azgael

**Comment by paradajz** [comment]
I am glad you like the project!

Support for MCP23017 isn't planned at the moment. There are multiple other options for I/O, however.
